cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/sh
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export AWS_SHARED_CREDENTIALS_FILE=/usr/local/e2e-aws-cluster-profile/.awscred
```

Comment:
Since this is dependent on the place where the secret is mounted in the job, could we set this as an env var in the job? Or mount the awscred to the "normal" place that the AWS libraries look?

Reply:
Done.